backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -306,3 +310,28 @@ fieldset {
 .form-buttons {
   text-align: center;
 }
+
+.form-section {
+  @include make-row(0px);
+  margin: 10px 0px;
+  padding: 10px 0px;
+  background: very-light($color-3, 4);
+  border: 1px solid very-light($color-3, 32);
+}
+
+.text-field-addon {
+  position: absolute;
+  right: 25px;
+  bottom: 21px;
+  pointer-events: none;
+}
```

Comment:
Here, I think we should also rely [on the build in](https://v4-alpha.getbootstrap.com/components/input-group/#basic-example) `input-group-addon`, again for the reason of free documentation, but as we want to have those _add-ons_  styled a tad bit different, we can add a supplementary class, say 

```css
.input-group-addon.simple {
  background-color: transparent;
  border-left: 0;
}
```